README.md

```diff
@@ -36,56 +47,83 @@ RP_PRIVATE_HEX_KEY=850e54b92c6291a1ff7b8c3ef30e032571ed77c9e5c78b1cd6ee5fec4fea9
 AUTH_REQUEST_EXPIRES_AFTER_SEC=300
 MOCK_AUTH_RESPONSE=false
 ```
+
 (Exception for the IP address this is a valid configuration to test with.)
 
-#### Build & start
-clone the repository:
+From the root directory
+
 ```bash
-git clone git@github.com:Sphereon/onto-web-demo.git
+cd <workspace>/onto-web-demo
+yarn global add concurrently
+yarn global add ts-node
+yarn install-all
+yarn build-types
```

Comment:
yarn build-types gives me this error : 

src/auth-model.ts:21:10 - error TS2339: Property 'userName' does not exist on type 'AuthResponse'.

21     this.userName = userName;
            ~~~~~~~~